php/web/src/AppBundle/Entity/Character.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace AppBundle\Entity;
+
+use NoGame\Assertion;
+use Ramsey\Uuid\Uuid;
+
+final class Character
+{
+    /**
+     * @var Uuid
+     */
+    private $id;
+
+    /**
+     * @var Uuid
+     */
+    private $accountId;
+
+    /**
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @var string
+     */
+    private $normalizedName;
+
+    /**
+     * @var int
+     */
+    private $health;
+
+    /**
+     * @var int
+     */
+    private $experience;
+
+    /**
+     * @param Uuid $accountId
+     * @param string $name
+     * @param int $health
+     */
+    public function __construct(Uuid $accountId, string $name, int $health)
+    {
+        $this->id = Uuid::uuid4();
```

Comment:
So here id of character is generated but also in: https://github.com/norzechowicz/no-game/blob/master/nodejs/server/src/NoGame/Engine/Player.js#L31
It should be generated in one place and in other places should be passed as param in constructor IMO